proxy/__init__.py

```diff
@@ -0,0 +1,215 @@
+"""
+The proxy module provides the means to proxy images from other registry instances.
+Registries following the distribution spec are supported.
+"""
```

Comment:
It would probably make sense to move this into its own `proxy.py` file under the `util/` directory.

Reply:
`util` is just a bucket for anything we don't have a specific place for. The advice here[1] is aimed at go, but applies to any language.
[1] https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Reply:
Going to move the tests here but keep the proxy dir as is otherwise, as agreed verbally.